src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -2007,11 +2007,19 @@ instruct reduce_addL(iRegLNoSp dst, iRegL src1, vReg src2, vReg tmp) %{
   ins_pipe(pipe_slow);
 %}
 
-instruct reduce_addF(fRegF dst, fRegF src1, vReg src2, vReg tmp) %{
+// Distinguish two cases based on requires_strict_order
+// 1. Non strictly-ordered AddReductionVF/D. For example, AddReductionVF/D
+//    generated by Vector API. It is more beneficial performance-wise to do
+//    an unordered FP reduction sum (vfredusum.vs).
+// 2. Strictly-ordered AddReductionVF/D. For example, AddReductionVF/D
+//    generated by auto-vectorization. Must do an ordered FP reduction sum
+//    (vfredosum.vs).
```

Comment:
Nit: Can you leave an empty line after the code comment?

Reply:
> Nit: Can you leave an empty line after the code comment?

Fixed.